service/service.go

```diff
@@ -22,6 +22,7 @@ type Service struct {
 	Teams        []string
 	Doc          string
 	IsRestricted bool `bson:"is_restricted"`
+	Instances    []ServiceInstance
```

Comment:
I don't think there's much else to review. With Andrews' suggestion, most changes move to the client, that will now need to issue to API calls and combine the results.

So, you will:

1. remove the service doc handler (or convert it into a handler that returns full information of the service)
1. add a handler for listing instances of a service (that the user has access to, obviously)

And then comes the client changes.

Thank you!